### DISCUSSION TITLE
add `capture` props to  FileInput component

### DISCUSSION THREAD

**Discussion by igorovic** [mantinedev/mantine#3261] [Feature requests]
A input element of type `file` supports a `capture` attributes which allow the user to provide a file directly from it's camera.

```html
<input type="file" id="imageFile" capture="user" accept="image/*" />
```

Is it possible to add a prop to set this attribute ? 

Here is the documentation: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture

**Comment by rtivital** [comment]
I'll add this feature in one of the next releases
